articles/definitions-time/index.zh-hans.html

```diff
@@ -0,0 +1,218 @@
+  </aside></div>
+  </section>
+
+
+  <section id="floating_time">
+  <h2>浮选时间</h2>
+  <p>有些时间值并不能映射到<a href="incremental_time" class="termref">累计时间</a>中的某个特定时刻。与之相反，它们仅仅代表一个名义时间，在世界各地的所有时区都会以相同的方式呈现。</p>
+  <p>举个例子，2016年6月11日星期六是英国女王官方的90岁生日。由于不同国家的时钟设置差异，6月11日在英国开始或结束的具体时间，在其他国家实际上有可能是周五或者周日了，但该事件的日期始终是6月11日星期六。</p>
+  <p>由于没有对应明确的<a href="#incremental_time" class="termref">累计时间</a>，因此，我们通常称此类时间为<dfn id="def_floating_time">浮选时间</dfn>。当绑定上对应的时区信息后，浮选时间会生成<em>一系列可被接收的累计时间的值</em>。</p>
+  <p>浮选时间不会也不应该归属于某个特定的时区。</p>
+  <p>举一些其他浮选时间事件的例子，比如：报纸的出版日期、里约奥运会的开始日期、新年开始的时间、办公时间设置为“9点到5点”等等，均不需要考虑<a href="#time_zone">时区</a>。</p>
+  </section>
+
+
+<section id="time_zone">
+  <h2>时区</h2>
+  <p><dfn id="def_time_zone">时区</dfn>是一组用于确定本地时间（也叫时钟时间）的规则，它与特定地理区域的累计时间（在大多数的计算系统中使用）相关。</p>
+  <p>如果要关联到时钟时间，时区规则中必须要考虑<a href="#time_zone_offset" class="termref">时区偏移量</a><em>和</em>任何<a href="#dst" class="termref">夏令时</a>的变更。</p>
+  </section>
+
+
+<section id="zone_offset">
+  <h2>时区偏移量</h2>
+  <p><dfn id="def_zone_offset">时区偏移量</dfn>是一个数量，根据世界各地相对于本初子午线的位置，基于<a href="utc" class="termref">UTC时间</a>进行加减得到。通常偏移量以一小时为间隔单位，但也可以存在不同，比如30或45分钟。</p>
+  <p>通常，在展示时间格式中使用+/-后跟偏移量，来表示时区偏移量。举个例子，日本比UTC时间早9小时，看到的时间会被显示为：<samp>2016-06-11 05:10+09:00</samp>。</p>
+  <p>需要特别注意的是，时区偏移量没办法将时间转换为<a href="#dst" class="termref">夏令时</a>强制生效的时钟时间。</p>
+  </section>
+
+
+ <section id="dst">
+  <h2>夏令时</h2>
+  <p><dfn id="def_dst">夏令时</dfn> (DST) 或者“夏季时间”的目的，主要是为了让人们在晚上有更多日照时间。</p>
+  <p>夏令时在不同国家是不同的（甚至一个国家部分地区也不一样），并且常常会由于一些特殊事件，而进行一次性的修改。同时，并非所有地区都遵守夏令时，通常靠近赤道的地区不需要夏令时的。</p>
+  <p>在做时间转换时，了解当地什么时间引入夏令时，什么时间废除夏令时，以及夏令时什么时间开始，什么时间结束（每年可能有所不同），是非常重要的。</p>
+  <p>举个例子，韩国标准时间和日本标准时间当前使用了相同的<a href="#zone_offset" class="termref">时区偏移量</a>，也均未使用夏令时。然而，日本是在1951年废弃了夏令时，韩国在1988年废弃掉，因此，如果应用程序需要追溯当时的时间值，需要使用当时的时区信息。</p>
+  </section>
+
+
+  <section id="time_zone_ids">
+  <h2>时区标识</h2>
+  <p><dfn id="def_time_zone_id">时区标识</dfn>可以标识出当前时间与<a href="utc" class="termref">UTC</a>的特定差异，包括<a href="#time_zone_offset" class="termref">时区偏移</a>和<a href="#dst" class="termref">夏令时</a>。</p>
+  <p>最权威的时区规则集是<a href="https://www.iana.org/time-zones">TZ database</a>（也称为奥尔森时区数据库）, 他被众多系统使用，比如各种商业UNIX操作系统、Linux操作系统、Java语言、CLDR数据集、ICU组件，以及其他众多的操作系统和组件库。但也存在其他时区规则集，比如Microsoft Windows使用自己的时区数据集和标识符。</p>
+  <p>在TZ database中，时区的ID通常由地区和示例城市组成，示例城市是相关时区中人们所熟知的典型城市。比如，美国太平洋时区在TZ database中的ID是<code class="kw" translate="no">America/Los_Angeles</code>。同时，TZ database还为许多ID提供了别名， 比如<code class="kw" translate="no">Asia/Ulan Bator</code>也可以称为<code class="kw" translate="no">Asia/Ulaanbaatar</code>.</p>
+  <p>CLDR（通用本地数据集 Common Locale Data Repository的英文首字母缩写）可以为ID提供本地化的形式。需要注意，部分系统（比如苹果的Mac OS）会提供额外的示例城市。</p>
+  </section>
+
+
+  <section id="calendars">
+  <h2>公历和其他日历</h2>
+  <p><dfn id="def_gregorian_calendar">公历</dfn>是使用最广泛的历法。它是太阳历, 一年通常365天，包含“闰年”的概念。每4年为闰年，增加一天，但当年份可以被100整除时，不用增加（同时，如果年份可以被400整除，仍需要增加）。</p>
+
+  <p>世界各地还有许多其他历法。其中一部分是月亮历；还有一部分基于公历，但开始时间不一样；还有一些历法随着某些重要人物的死亡而重置。通常这些历法用于宗教目的，但有时也被用于报纸和电子邮件，也可以是出生日期。</p>
```

Comment:
```suggestion
  <p>世界各地还有许多其他历法。其中一部分是阴历；还有一部分基于公历，但开始时间不一样；还有一些历法随着某些重要人物的死亡而重置。通常这些历法用于宗教目的，但有时也被用于报纸和电子邮件，也可以是出生日期。</p>
```

lunar calendar一般译为[阴历](https://zh.wikipedia.org/wiki/%E9%98%B4%E5%8E%86)。